omnibus/config/software/chef-dk.rb

```diff
@@ -79,23 +79,14 @@
   appbundle "test-kitchen", lockdir: project_dir, gem: "test-kitchen", without: %w{changelog debug docs provisioning}, env: env
   appbundle "inspec", lockdir: project_dir, gem: "inspec", without: %w{deploy tools maintenance integration}, env: env
 
-  %w{chef-apply chef-dk chef-vault ohai opscode-pushy-client cookstyle dco berkshelf}.each do |gem|
+  %w{chef-dk chef-apply chef-vault ohai opscode-pushy-client cookstyle dco berkshelf}.each do |gem|
     appbundle gem, lockdir: project_dir, gem: gem, without: %w{changelog}, env: env
   end
 
-  # Clear the now-unnecessary git caches, cached gems, and git-checked-out gems
-  block "Delete bundler git cache and git installs" do
+  # Clear git-checked-out gems (most of this cleanup has been moved into the chef-cleanup omnibus-software definition,
+  # but chef-client still needs git-checked-out gems)
+  block "Delete bundler git installs" do
     gemdir = shellout!("#{install_dir}/embedded/bin/gem environment gemdir", env: env).stdout.chomp
-    remove_directory "#{gemdir}/cache"
     remove_directory "#{gemdir}/bundler"
   end
-
-  # Clean up docs
-  delete "#{install_dir}/embedded/docs"
```

Comment:
This all moved to chef-cleanup i assume?

Reply:
Yes